frontend/src/component/admin/apiToken/ApiTokenForm/TokenTypeSelector/TokenTypeSelector.tsx

```diff
@@ -57,36 +36,39 @@ export const TokenTypeSelector = ({
                     defaultValue="CLIENT"
                     name="radio-buttons-group"
                     value={type}
-                    onChange={(event, value) => setType(value)}
+                    onChange={(event, value) => setType(value as TokenType)}
                 >
-                    {selectableTypes.map(({ key, label, title }) => (
-                        <FormControlLabel
-                            key={key}
-                            value={key}
-                            sx={{ mb: 1 }}
-                            control={
-                                <Radio
-                                    sx={{
-                                        ml: 0.75,
-                                        alignSelf: 'flex-start',
-                                    }}
-                                />
-                            }
-                            label={
-                                <Box>
+                    {apiTokenTypes.map(
+                        ({ key, label, title, permission, hasAccess }) => (
```

Comment:
Unused `permission`

Reply:
After removing permission this got simplified: [6d67940](https://github.com/Unleash/unleash/pull/4072/commits/6d67940d64d9c7d24d44faf360d5fc9aad061b24)